scripts/etc/udhcpc/default.script

```diff
@@ -19,7 +19,7 @@ PEERDNS_IF=eth0
 case "$1" in
   deconfig)
     # bring interface up, but with no IP configured:
-    ip addr flush dev $interface
+    ip -f inet addr flush dev $interface
     ip link set $interface up
```

Comment:
I should probably experiment with it myself (some more), but you may already know it by heart:
why does this line not have a `-f inet`?

Otherwise, it's looking very good and I will merge your PR today